src/DetailsView/components/narrow-mode-detector.tsx

```diff
@@ -31,7 +32,8 @@ export function getNarrowModeComponentWrapper<P>(
         const isNarrowModeEnabled = props.isNarrowModeEnabled === true;
         const narrowModeThresholds = props.deps.getNarrowModeThresholds();
 
-        const isNarrowerThan = (threshold: number) => dimensions.width < threshold;
+        const isNarrowerThan = (threshold: number | undefined) =>
```

Comment:
This change was necessary due to typings changing and our linter (rightfully) not allowing it. I didn't want to do more than this because I think the narrow mode detector could likely be refactored to be a bit better in the near future and decided to make changes as required.